modelskill/metrics.py

```diff
@@ -533,18 +533,20 @@ def explained_variance(obs: np.ndarray, model: np.ndarray) -> float:
 def pr(
     obs: np.ndarray,
     model: np.ndarray,
+    time: pd.DatetimeIndex,
```

Comment:
doesn't it seem a bit odd to have time as a separate arg, instead of passing obs and model as series 🤔?